platform/innovium/sonic-platform-modules-cameo/esc601-32q/modules/pmbus.h

```diff
@@ -0,0 +1,425 @@
+/*
+ * pmbus.h - Common defines and structures for PMBus devices
```

Comment:
Why not use the pmbus.h which is part of the Linux kernel?

Reply:
The pmbus.h is useless for the platform driver. Removed.